### ISSUE TITLE
Linux: No DLL generated on /compile or /pcompile

### ISSUE THREAD

**Issue by Siriusmart** [ClassiCube/MCGalaxy#810]
## Setup

- MCGalaxy version: git latest ([da4785f](https://github.com/ClassiCube/MCGalaxy/commit/da4785fda27b3ba5ccfc63421e300d7bbf40f79c))
- dotnet version: 8.0.107
- mono version: 6.12.0

The server is compiled from /CLI because the release zip file only contains a .exe. The compiled server runs right and there doesn't seem to be an issue with it.

## Expected behaviour

When ran `/compile`, a .dll file should pop up in the dll/ directory, and the command can be loaded.

## Observed behaviour

No .dll file is generated, as such the command cannot be loaded.

After a bit of poking around, the behaviour is the same with
- The hello world command i wrote.
- A random command I took from MCGalaxy-Plugins.
- A random plugin I took from MCGalaxy-Plugins.

## Attachments

```
/compile HelloClassiCube
(22:41:13) Command compiled successfully from CmdHelloClassiCube.cs
(22:41:13) (console) used /Compile HelloClassiCube
/cmdload HelloClassiCube
(22:41:18) File extra/commands/dll/CmdHelloClassiCube.dll not found.
(22:41:18) (console) used /CmdLoad HelloClassiCube
```

https://github.com/user-attachments/assets/6981a432-badb-4721-bb71-75742c3e08b2

---

Thank you in advance.

**Comment by Goodlyay** [comment]
I don't know how to solve this at all but this is a really well formatted issue report, just had to say that

**Comment by Siriusmart** [comment]
Resolved in latest commit ([3ff3089](https://github.com/ClassiCube/MCGalaxy/commit/3ff3089589d26ddebfc2370e4d5a5efff7a28820))

![image](https://github.com/user-attachments/assets/8aa4aed0-6932-4e63-9db0-e0322281ed10)

(also MCG_DOTNET_PATH should be the binary in `/usr/share/dotnet/dotnet` instead of the symlinked `/usr/bin/dotnet`, which was a bit confusing at first)
